VaultOfTheIncarnates/BroodkeeperDiurna.lua

```diff
@@ -188,6 +197,19 @@ function mod:AddMarking(_, unit, guid)
 	end
 end
 
+function mod:AddMarking2(_, unit, guid)
```

Comment:
See above comment, this needs to be merged with AddMarking so they can work along eachother.